### DISCUSSION TITLE
Renew Copilot Subscription

### DISCUSSION THREAD

**Discussion by mallikarjunrterdal** [community/community#83457] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My GitHub Copilot subscription is ending in next 2 days and I'm unable to find the renew option. May I know how to renew my subscription?

Additional information: I had clicked Cancel button as I was not sure if I will be involving in coding. But now, I am involving in coding

**Comment by vitoff34** [comment]
Did you find the solution? I am struggling as well..

**Reply by davecheney** [reply, marked as answer]
@vitoff34 you are currently in your 30 day trial period; when that ends (apparently tonight) you will automatically be billed every month for Copilot. No action is required.

**Comment by Immu123456789** [comment]
I have upgraded copilot but unable to use in vscode

**Comment by SADAAN-UL-HAQUE** [comment]
I have a GitHub Copilot student account that will expire in 2 days. How can I find out if I have not received any email or notification?

**Comment by Hyzyr** [comment]
I need to cancel left 5 days of free trial and I want to start new pro subscriotion but I dont have that option, only option I have at the moment is to upgrade to pro+ which is little expensive

**Comment by lakith93** [comment]
I have subscribed copilot pro+ in 08th Feb 2026, it should be renew on 08th March 2026. But it tried and remove my privileges. But why?
